js/main.js

```diff
@@ -129,9 +217,13 @@ function renderMustSee() {
         deleteModalAppear(event);
       });
       const favoriteButton = document.createElement('button');
+      favoriteButton.setAttribute('value', mustSeeData[i].id);
       favoriteButton.setAttribute('type', 'submit');
       favoriteButton.classList.add('page-font', 'mt-1', 'btn-warning', 'btn-lg');
       favoriteButton.textContent = 'Make Favorite';
+      favoriteButton.addEventListener('click', () => {
+        favoriteModalAppear(event);
+      });
```

Comment:
Make sure to capture the actual event and not use the global event:

```JAVASCRIPT
favoriteButton.addEventListener('click', (event) => {
  favoriteModalAppear(event);
});
```

Also since you are just passing the same parameter through you can just pass the function without the arrow function:

```JAVASCRIPT
favoriteButton.addEventListener('click', favoriteModalAppear);
```